packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorModule.m

```diff
@@ -112,6 +112,7 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
                       }
                   });
       }}];
+    CGImageRelease(cgImage);
```

Comment:
Does `finalImage` depend on `cgImage` being present? In other words, could we `CGImageRelease(cgImage)` earlier in code? Moving it to line 91 would make it clear that there is no race condition happening (consuming `finalImage` in the handler vs releasing `cgImage`). If `finalImage` depends on `cgImage` we would be better of releasing the image inside the `detectFromImage:` callback.